src/LinuxTracing/LibunwindstackUnwinder.cpp

```diff
@@ -67,20 +67,13 @@ class LibunwindstackUnwinderImpl : public LibunwindstackUnwinder {
  public:
   LibunwindstackResult Unwind(pid_t pid, unwindstack::Maps* maps,
                               const std::array<uint64_t, PERF_REG_X86_64_MAX>& perf_regs,
-                              const void* stack_dump, uint64_t stack_dump_size) override;
+                              const void* stack_dump, uint64_t stack_dump_size, bool offline_memory,
```

Comment:
The default value of `offline_memory` is not specified here, while the default value of `max_frames` is. I guess they should both be specified (or none?)?